apis/supernova/src/handler/hypercube-handler.js

```diff
@@ -109,7 +109,7 @@ class HyperCubeHandler extends DataPropertyHandler {
 
     super.setProperties(properties);
 
-    this.hcProperties = this.path ? utils.getValue(properties, `${this.path}.qHyperCubeDef`) : properties.qHyperCubeDef;
```

Comment:
I think '.qHypercubeDef' is needed when it is used in sense-client, so I might need to add a flag and keep it anyway for when the flag is false.